machine.go

```diff
@@ -296,6 +291,12 @@ func NewMachine(ctx context.Context, cfg Config, opts ...Opt) (*Machine, error)
 	m.machineConfig = cfg.MachineCfg
 	m.Cfg = cfg
 
+	m.Handlers.FcInit = m.Handlers.FcInit.Swap(NewSetupNetworkHandler(m.netNSPath()))
+
+	for _, opt := range opts {
```

Comment:
Why'd you want to move the opts down here?

Reply:
I moved them down because that's generally where they belong. They usually are towards the end of a function, but I found that we were adding things to the `NewMachine` function which could stomp over any option that was provided. So I moved it down for safety purposes to avoid stomping over user overrides.

Reply:
Can you do that in a separate commit?  It's unrelated to the subject of this commit (and PR).

Reply:
Just removed altogether. I'll go ahead and create a separate PR for this.